src/Symfony/Component/Security/Core/Authentication/RememberMe/PersistentTokenInterface.php

```diff
@@ -21,13 +21,6 @@
  */
 interface PersistentTokenInterface
 {
-    /**
-     * Returns the class of the user.
-     *
-     * @return string
-     */
-    public function getClass();
```

Comment:
should be `@deprecated` instead

Reply:
I've removed this.. the tests broke and I'm not even sure they should get changed. I'm not understanding a lot at this moment to be honest.